client/components/mma/shared/ProgressIndicator.tsx

```diff
@@ -9,7 +9,7 @@ interface Step {
 }
 
 interface ProgressIndicatorProps {
-	steps: [Step, Step, Step];
+	steps: [Step, Step, ...Step[]];
```

Comment:
This isn't `Step[]` because there are guaranteed to always be _at least_ 2 steps?

Reply:
yes exactly, I wanted it to be an array of steps with some guards around the number of elements